adding closeOnTap option to FocusedMenuItem

dimzeta opened the pull request:
Fixes #25 

I have added `closeOnTap` option on FocusedMenuItem.
Default value to true.

If it is set to false, the menu is not closed when you click on the FocusedMenuItem.

It is useful when you have a "Favorites" menu item, as in the example.
I also implemented it in the example:

![Alt Text](https://s9.gifyu.com/images/RPReplay_Final1631717497.gif)

henriquezanfa commented:
Hey @dimzeta    thanks for your PR
There are some conflicts with master, can you fix them?

dimzeta commented:
Hello @henriquezanfa,

Since the code is old, I created another PR to fix some bugs and null safety errors: https://github.com/retroportalstudio/focused_menu/pull/43

I updated the code based of this PR.

I also added a `FocusedMenuItem` with `closeOnTap: false` in the example.

henriquezanfa commented:
Since you created another pull request I'm closing this and we can continue the discussion there.
